### PR TITLE
Create git tips

### PR DESCRIPTION
**Pull request by nhcarrigan**
Signed-off-by: nhcarrigan <nhcarrigan@gmail.com>

closes #ISSUE-NO

#### What does this PR do?

Creates a `tips` folder and adds a `git.md` file for git-related tips.

#### Description of Task to be completed?

(I'm actually not sure what goes here...)

#### How can this be manually tested?

Read it! 😁 
- Confirm that the commands are correct and work as intended
- Definitions are clear and accurate
- Steps can be followed in an actual project.

#### Any background context you want to provide?

Came from a discussion with @eddiejaoude and @Vyvy-vi on the Discord server.

#### What is the relevant issue link?

There isn't one. 🙂

### CODE REVIEW

**Review by Vyvy-vi — APPROVED** [review]

File: tips/git.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
+# `git` Related Tips
+
+This file contains various tips and tricks for using the `git` CLI (and GitHub too!)
+
+## Glossary
+
+Some helpful `git` and GitHub related terms.
+
+- `repository`: A repository is like a folder that holds a project's files, but unlike other folders a repository is tracked for changes by `git`.
```
Comment (AkshayaKulasekaran) [inline tips/git.md:9]: Might want to mention that `repo` is a short form of `repository` or vice versa for future convinience

File: tips/git.md [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+
+## Glossary
+
+Some helpful `git` and GitHub related terms.
+
+- `repository`: A repository is like a folder that holds a project's files, but unlike other folders a repository is tracked for changes by `git`.
+- `commit`: A commit holds the specific changes to files that were made since the last `commit`.
+- `PR`: Short for "Pull Request", this is a request to merge the changes made on one repository or branch into another repository or branch.
+- `branch`: A branch is like a new workspace for the same project, allowing you to work on changes, commit them, and push them to a repository without affecting the original branch.
+- `fork`: A forked repository is a "copy" of a repository that has references to the original. GitHub will track differences between a fork and a source for you, and show the number of different commits.
```
Comment (AkshayaKulasekaran) [inline tips/git.md:13]: Might want to add `clone` and the difference between forking and cloning

**Review by AkshayaKulasekaran — CHANGES_REQUESTED** [review]
I think it looks good! Just added two suggestions for clarity

**Comment by nhcarrigan** [comment]
@AkshayaKulasekaran I've added the terms you requested - are the definitions clear enough? 🙂

**Comment by AllanRegush** [comment]
Where is the original discussion for this change?
I don't have my context for this. I'll leave this for @eddiejaoude.

**Review by AllanRegush — APPROVED** [review]
Looks Good :+1: Thank You

**Comment by eddiejaoude** [comment]
Great team effort, thank you all 👍

**Pull request merged**